tests/tools/test_dataframe_html_output.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+import pytest
+
+
+@pytest.fixture()
+def df_with_urls():
+    pd.set_option("display.notebook_repr_html", True)
+    data = {"col1": [1, 2], "col2": ["http://foo.com", "https://bar.com"]}
+    return pd.DataFrame(data)
+
+
+def test_df_has_clickable_urls(df_with_urls):
```

Comment:
The convention we use is `test_function\class_description`.

Like `test_df_repr_html_true`

Does it look better? Mainly it's done for quick search.